kit/typing.py

```diff
@@ -3,6 +3,8 @@
 
 __all__ = ["implements"]
 
+from typing_extensions import get_type_hints
```

Comment:
```suggestion
from typing import get_type_hints
```

Reply:
isn't PalKit for 3.7+?

Reply:
yes? `get_type_hints()` has been in the standard library since python 3.5

Reply:
maybe you're confusing it with `get_args()` or `get_origin()` which were added in 3.8, I think.